package.json

```diff
@@ -13,7 +13,8 @@
   "scripts": {
     "lint": "standard",
     "test": "ava --no-color test/*.js",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "commit": "git cz"
```

Comment:
Doesn't this depend upon having this alias in your .gitconfig?

Reply:
Perhaps but from what I know and experienced it's set automatically (just running `npm i -g comitizen` handles that).
I might be wrong tho but after having installed it on WSL on my work laptop, I didn't had to configure anything.

Reply:
Seems to work. You should add a note to the Contribution guide about what this is and how it works. Could you?

Reply:
Sure, I'll do that now.